Source/Core/Core/PowerPC/Jit64Common/EmuCodeBlock.cpp

```diff
@@ -269,6 +273,18 @@ class MMIOReadCodeGenerator : public MMIO::ReadHandlingMethodVisitor<T>
       if (m_sign_extend)
         m_code->MOVSX(32, sbits, m_dst_reg, R(m_dst_reg));
     }
+    if (swap && (sbits != 8))
+    {
+      m_code->BSWAP(64, m_dst_reg);
+      switch (sbits)
+      {
+      case 16:
+        m_code->SHR(64, R(m_dst_reg), Imm8(48));
+        break;
+      case 32:
+        m_code->SHR(64, R(m_dst_reg), Imm8(32));
```

Comment:
Should also be given a break, even if it's the last switch case.

Reply:
Done.